### PR TITLE
NVENC AV1 Encoding Support

### PR DESCRIPTION
**Pull request by NotJansel**
I started working on AV1 Encoding Support for danser, currently only tested with NVENC. I may get back to this later with the other encoder options, but feel free to give opinions and feedback to this.

### CODE REVIEW

File: tools/ffmpeg/ffmpeg.go [hunk 1]
```diff
@@ -15,12 +15,12 @@ import (
...
 
-	archiveName := "ffmpeg-n5.1-latest-linux64-gpl-shared-5.1"
+	archiveName := "ffmpeg-n6.1-latest-linux64-gpl-shared-6.1"
 	if runtime.GOOS == "windows" {
-		archiveName = "ffmpeg-n5.1-latest-win64-gpl-shared-5.1"
+		archiveName = "ffmpeg-n6.1-latest-win64-gpl-shared-6.1"
 	}
 
-	downloadUrl := "https://github.com/Wieku/FFmpeg-Builds/releases/download/latest/" + archiveName + ".zip"
+	downloadUrl := "https://github.com/BtbN/FFmpeg-Builds/releases/download/latest/" + archiveName + ".zip"
```
Comment (Wieku) [inline tools/ffmpeg/ffmpeg.go:23]: Please don't relink to BtBN builds, there's a reason why custom builds are used (smaller size as we don't need all encoders/decoders). 

Most recent [build](https://github.com/Wieku/FFmpeg-Builds/releases/tag/latest) should have av1_nvenc support added.

**Review by Wieku — CHANGES_REQUESTED** [review]

File: tools/ffmpeg/ffmpeg.go (hunk 1 above)
Comment (NotJansel) [reply tools/ffmpeg/ffmpeg.go:23]: ah i see, will get to it later (I linked to it because the custom builds didnt have av1 before)

**Comment by NotJansel** [comment]
image
After changing to the custom build, it seems it broke for any encoder...

**Comment by Wieku** [comment]
@NotJansel redownload, it should work now

**Comment by NotJansel** [comment]
i needed to manually download rn, as the autobuild published a file with a different name, but it works finally
image

**Comment by NotJansel** [comment]
so as everything works, ill leave this pr as just for nvenc av1 encoding

**Review by Wieku — APPROVED** [review]

**Pull request merged**